# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsChild.h
# Commit: 43dbad71664b
# Full Hash: 43dbad71664bb2c5779b54decd5133db076ec6a0
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-08 21:36:09
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Replace pseudo-move constructor of StructuredCloneReadInfo by explicit DeserializeStructuredCloneReadInfo function. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46593
# ==============================================================================

diff -r 19188ca45113 -r 43dbad71664b dom/indexedDB/ActorsChild.h
--- a/dom/indexedDB/ActorsChild.h	Fri Nov 08 14:28:39 2019 +0000
+++ b/dom/indexedDB/ActorsChild.h	Fri Nov 08 13:22:08 2019 +0000
@@ -726,9 +726,6 @@
 
   void HandleResponse(const nsTArray<IndexCursorResponse>& aResponses);
 
-  StructuredCloneReadInfo PrepareCloneReadInfo(
-      SerializedStructuredCloneReadInfo&& aCloneInfo) const;
-
   template <typename T, typename Func>
   void HandleMultipleCursorResponses(const nsTArray<T>& aResponses,
                                      const Func& aHandleRecord);